tests/shadow/test_shadow_measurement.py

```diff
@@ -165,7 +142,7 @@ def test_measurement_process_copy(self, wires, seed):
     @pytest.mark.parametrize("shots", shots_list)
     @pytest.mark.parametrize("seed", seed_recipes_list)
     @pytest.mark.parametrize("interface", ["autograd", "jax", "tf", "torch"])
-    @pytest.mark.parametrize("device", ["default.qubit", "default.mixed", None])
+    @pytest.mark.parametrize("device", ["default.qubit", "default.mixed"])
```

Comment:
See my previous comment about testing the default implementation.

Reply:
Changed back to original